github/github.py

```diff
@@ -80,6 +93,11 @@ def get_api_fqdn(github_repo):
 
 
 def query_github(api_url_template, github_repo):
+    resp = raw_query_github(api_url_template, github_repo)
+    return resp.payload
+
+
+def raw_query_github(api_url_template, github_repo):
```

Comment:
Do you want the raw response returned everywhere and all code that uses web requests to be modified then?